specs/merge/validator.md

```diff
@@ -62,9 +62,11 @@ def get_execution_payload(state: BeaconState) -> ExecutionPayload:
             return ExecutionPayload()
         else:
             # Signify merge via producing on top of the last PoW block
-            return produce_execution_payload(pow_block.block_hash)
+            timestamp = compute_time_at_slot(state, state.slot)
```

Comment:
What does the execution-layer use this `timestamp` for?

Reply:
There is a `TIMESTAMP` opcode it is used for. Also, the difference of timestamp between child and parent block is enforced by block validity checks and difficulty formula, though, these checks will be omitted after the merge. But we have to support the opcode.

Reply:
Got it. thanks!